### DISCUSSION TITLE
最新版本好像最高画质可选项只有1080p

### DISCUSSION THREAD

**Discussion by yyz0313** [ywmoyue/biliuwp-lite#479] [Q&A] [answered]
我一开始好奇是不是我没开会员，然后发现最高画质确实就只有1080p

**Comment by ywmoyue** [comment]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/f6a49774-9f58-4181-9ec7-871057da0d8a)
 有4K呀，你试了哪个视频不行

**Reply by ywmoyue** [reply]
不过好像HDR选项出问题了

**Reply by yyz0313** [reply]
好像都不太行。。

**Reply by yyz0313** [reply]
![4a4a404517c22534949f115d8d87326](https://github.com/ywmoyue/biliuwp-lite/assets/103877997/50e876a8-9d1e-4da0-9375-def4d0df0746)
我试了你的那个视频很奇怪。我换了视频编码也不可以？我该怎么排除 求

**Reply by GD-Slime** [reply]
> 不过好像HDR选项出问题了

我这里倒是有hdr 就是没显示的设备 导致过曝了

**Comment by ywmoyue** [comment]
> > 不过好像HDR选项出问题了
> 
> 我这里倒是有hdr 就是没显示的设备 导致过曝了

是的，我忘了启用HDR要切换H265

**Comment by ywmoyue** [comment, marked as answer]
@yyz0313 重新登录后再试一试

**Reply by yyz0313** [reply]
重新登陆以后可以了！感谢！我想问一下4k清晰度是否跟漫游一样不需要大会员！！

**Reply by ywmoyue** [reply]
4k应该是需要大会员的，这里用的是官方接口，官方能看就能看，不能看就看不了

**Reply by yyz0313** [reply]
有没有考虑加入破解功能？

**Reply by ywmoyue** [reply]
不考虑

**Comment by yyz0313** [comment]
老哥抱歉，我没在软件里找到查看日志的地方，而且我看好像播放设置里也没有硬解选项。。好奇怪
